<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>514</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>630</width>
    <height>514</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>630</width>
    <height>514</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btn_upload_data">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>601</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Показать посты выбранной рубрики</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_save_data">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>420</y>
     <width>331</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Сохранить выбранную статью в файл</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>405</y>
     <width>611</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>611</width>
     <height>271</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>300</number>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>300</number>
   </attribute>
   <column>
    <property name="text">
     <string>Заголовок</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Краткое описание</string>
    </property>
   </column>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>601</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Новая Эпоха и Новый Мировой Порядок</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>О вере</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Мир сегодня</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Здоровье</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Пророчества Библии</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Из потустороннего</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Рецепты</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Разное</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Поделки</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>История древних цивилизаций</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Смешное</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Израиль</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Как работает Дух Святой</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Музыка</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Чудаки и необычное</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Поучительные истории</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Советы по выживанию</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Эволюция или Создание</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Полезности</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Споры с православными</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Монстры в истории</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Чудеса в нашей жизни</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Книга &quot;Избушка&quot;-Встреча с Богом</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Психология</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ислам</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Христианские праздники</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Вязание</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Книга Е.Самойловой &quot;В приближении 5-ой тру</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Книга &quot;Дети Падших.Как История стала Мифом&amp;qu</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Украина</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Потомки Хама</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Неоязычество</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Книга &quot;Се, стою у двери и стучу&quot;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Валяние</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_selected">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>601</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Выбрана рубрика:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_save_all">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>611</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить все статьи</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>611</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
